src/components/molecules/CLICommands/CLICommands.tsx

```diff
@@ -36,46 +37,53 @@ const testSuiteScripts: CLIScript[] = [
   {
     label: 'Run test suite',
     command: (name: string) => `kubectl testkube run testsuite ${name}`,
+    filename: 'run-test-suite.sh',
   },
   {
     label: 'Delete test suite',
     command: (name: string) => `kubectl testkube delete testsuite ${name}`,
+    filename: 'delete-test-suite.sh',
```

Comment:
all this props filename should be removed in future, after we create a function that transforms ```This is my test``` into ```this-is-my-test```.
I think we can use it in this case, basically transforming label to filename, adding by default ```.sh``` after.

Reply:
We don't need them. Thanks for spotting it. We got the name directly from the url that is generated for the entity.